controllers/cluster_controller_test.go

```diff
@@ -173,6 +170,8 @@ func TestClusterReconcilerReconcileSelfManagedCluster(t *testing.T) {
 			ReconciledGeneration: 1,
 		},
 	}
+	eksaRelease := test.EKSARelease()
+	bundles := createBundle()
```

Comment:
i recommend moving the instantiation of these variables closer to where they are used.